content/300-accelerate/250-connection-pooling.mdx

```diff
@@ -4,39 +4,56 @@ metaTitle: 'Accelerate: Connection Pooling'
+:::
 
-</Admonition>
+Accelerate provides built-in connection pooling by default. By using Accelerate, you get the benefits of connection pooling without having to configure anything. The efficient management of database connections allows the database to process more queries without exhausting the available database connections, making your application more scalable.
+
+In some cases, however, it may be beneficial to further configure connection pooling in order to optimize the performance of your application.
 
-### Default connection pool size
+### Connection Pooling in Accelerate
 
-Accelerate has a default `connection_limit` set to `10`.
+By default, Accelerate has a default connection limit of `10`. In most cases, this connection limit is sufficient. However, in high traffic or compute environments, you may need to [increase this value](#configuring-the-connection-pool-size). For the [Pro plan](https://prisma.io/pricing) there is a maximum connection limit of `100`. For [Business plan](https://prisma.io/pricing) users the maximum is `1000`.
 
 ### Configuring the connection pool size
 
-The connection pool size can be configured via the _database connection string_ in your Platform project (_not_ the Accelerate connection string in your Prisma schema) via the `connection_limit` parameter. For example, set a connection pool size of `10` like this: `postgresql://USER:PASSWORD@HOST:PORT/DATABASE?connection_limit=10`.
+The connection pool size can be configured in your Platform project via the `Connection limit` setting. For example, set a connection pool size of `10` like this:
+
+![Update database connection string in Accelerate](/img/accelerate/accelerate-update-database-connection-string.png)
 
-If nothing is configured, a default value will be used.
+If nothing is configured, the default value of `10` will be used.
 
 To change the size of the connection pool:
 1. Open the [Console](https://console.prisma.io/login).
 1. Select the _project_ in which you're using Accelerate. You might need to select a different _workspace_ if you can't see your project.
 1. Select the _environment_ where you want to configure Accelerate's connection pool size.
-1. Update the connection string by appending the `connection_limit` argument. 
+1. Update the connection limit via the `Connection limit` setting.
 
 ![Update database connection string in Accelerate](/img/accelerate/accelerate-update-database-connection-string.png)
 
+#### Autoscaling
+
+Autoscaling allows Accelerate to allocate resources to scale up or down based on the traffic to your application. Specifically, Accelerate will begin allocating new resources when you have consistent usage greater than 90% of your currently alloted connection limit. If traffic continues to increase, Accelerate will scale up further. If traffic decreases, Accelerate will scale down.
```

Comment:
Reposting this as I think it got lost in one of the changesets.

> Specifically, Accelerate will begin allocating new resources when you have consistent usage greater than 90% of your currently alloted connection limit.

I'd prefer if we left out any specific details about scaling thresholds and focus more on how a higher connection limit enables Accelerate to process more requests concurrently. Specific details might introduce friction if we want to refine these values later. I also don't think it provides much values to the user as they just want to understand what options they have for supporting more queries and the tangible option is the connection limit. It's our job to make sure the resources we allocate can support the connections they've given us.